src/shared/components/DownloadControls.tsx

```diff
@@ -17,6 +17,7 @@ interface IDownloadControlsProps {
     additionalLeftButtons?:ButtonSpec[],
     additionalRightButtons?:ButtonSpec[]
     dontFade?:boolean;
+    dontDisable?:boolean;
```

Comment:
why would we want to have the buttons not be disabled if the svg isnt available?

Reply:
I thought it had a dependency on having a `<svg ref={this.svgRef}` wrapper, but forgot that `<VictorySelectionContainer containerRef={this.svgRef}` also takes a function, so fixed.